src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceControllerTest.kt

```diff
@@ -483,8 +483,10 @@ class LicenceControllerTest {
       probationLduCode = "LDU1",
       dateCreated = LocalDateTime.now(),
       createdByUsername = "X12345",
-      standardConditions = someStandardConditions,
-      additionalConditions = someAdditionalConditions,
+      standardLicenceConditions = someStandardConditions,
+      standardPssConditions = someStandardConditions,
+      additionalLicenceConditions = someAdditionalConditions,
+      additionalPssConditions = someAdditionalConditions,
```

Comment:
Will this result in the same type of standard and additional conditions on a licence? ie. not separated into different types?

Reply:
The licence object and the createLicence request object are agnostic of the `conditionType` field. It only splits them out into separate condition collections as they are here